.travis.yml

```diff
@@ -15,9 +15,6 @@ php:
   - hhvm
 
 matrix:
-  include:
-    - php: 5.3
-      env: deps=low
```

Comment:
@xabbuh I'm ok to remove it, but I can't manage to reproduce the error even with ```deps=low``` and the right versions of PHP and phpUnit. Have you any ideas of what might be causing travis build to fail?